libafl_qemu/src/asan.rs

```diff
@@ -446,21 +446,21 @@ where
 pub fn gen_readwrite_asan<I, QT, S>(
     hooks: &mut QemuHooks<'_, I, QT, S>,
     _state: Option<&mut S>,
-    pc: u64,
+    #[cfg(cpu_target = "arm")] pc: u32,
```

Comment:
pc here is u64, regardless of the arch

Reply:
or better, change it to GuestAddr

Reply:
Ohh I did not notice that a guest-specific type exists. I will change it with the next push